docs/src/main/asciidoc/spring-cloud-consul.adoc

```diff
@@ -149,21 +149,71 @@ spring.cloud.consul.discovery.management-suffix
 spring.cloud.consul.discovery.management-tags
 ----
 
-=== HTTP Health Check
+==== HTTP Health Check
 
-The health check for a Consul instance defaults to "/health", which is the default locations of a useful endpoint in a Spring Boot Actuator application. You need to change these, even for an Actuator application if you use a non-default context path or servlet path (e.g. `server.servletPath=/foo`) or management endpoint path (e.g. `management.server.servlet.context-path=/admin`). The interval that Consul uses to check the health endpoint may also be configured.  "10s" and "1m" represent 10 seconds and 1 minute respectively.  Example:
+The health check for a Consul instance defaults to "/actuator/health", which is the default location of the health endpoint in a Spring Boot Actuator application. You need to change this, even for an Actuator application, if you use a non-default context path or servlet path (e.g. `server.servletPath=/foo`) or management endpoint path (e.g. `management.server.servlet.context-path=/admin`).
+
+The interval that Consul uses to check the health endpoint may also be configured. "10s" and "1m" represent 10 seconds and 1 minute respectively.
+
+This example illustrates the above (see the `spring.cloud.consul.discovery.health-check-*` properties in link:appendix.html[the appendix page] for more options).
 
 .application.yml
 ----
 spring:
   cloud:
     consul:
       discovery:
-        healthCheckPath: ${management.server.servlet.context-path}/health
+        healthCheckPath: ${management.server.servlet.context-path}/actuator/health
         healthCheckInterval: 15s
 ----
 
-You can disable the health check by setting `management.health.consul.enabled=false`.
```

Comment:
This disables the ConsulHealthIndicator, not the HTTP check from Consul to the instance.